resolveVideos.py

```diff
@@ -39,19 +39,33 @@ def main(detected_faces_folder,reference_faces_file, tolerance):
         reference_faces = defaultdict(dict)
         cold_start = True
 
-    filtered_folder = [file for file in os.listdir(detected_faces_folder) if os.path.splitext(file)[1] in ['.pkl', \
-                                                                                                           '.pickle']]
+    # Load file name/content hash table if available
+    if os.path.isfile(in_hash_table):
+        hash_table = pickle.load(open(in_hash_table, "rb"))
+        print("Loaded hash table from {0} with {1} entries".format(in_hash_table, len(hash_table['hash_to_file'])))
+    else:
+        print("No hash table pickle file found at {0}, starting with blank table".format(in_hash_table))
+        hash_table = defaultdict(dict)
+
+    filtered_folder = [file for file in os.listdir(detected_faces_folder) if file[-20:] == "face_detected.pickle"]
```

Comment:
probably want to do
[file for file in os.listdir(...) if file.endswith('face_detected.pickle')]